README.md

```diff
@@ -5,27 +5,32 @@
 `wskdeploy` is currenty under development and in its very early stages.  Check back often to see how its progressing.
 
 # How to use
-`wskdeploy` is written in Go. You can invoke it as a Go program, or run its binary file `wskdeploy` directly after building.
+`openwhisk-wskdeploy` is written in Go. You can invoke it as a Go program, or run its binary file `openwhisk-wskdeploy` directly after building.
```

Comment:
Let's keep the name `wskdeploy` and instead add build instructions to produce that binary name:

`go build -o wskdeploy`